packages/react/src/PricingOptions/PricingOptions.tsx

```diff
@@ -28,6 +28,7 @@ import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/compone
 import styles from './PricingOptions.module.css'
 
 export type PricingOptionsProps = {
+  align?: 'start' | 'center'
```

Comment:
You could bring `AlignOptions` up a bit and use it here

```suggestion
  align?: AlignOptions
```